addons/web/static/tests/qunit.js

```diff
@@ -286,13 +286,18 @@ export function setupQUnit() {
             console.log("test successful");
         } else {
             console.error(errorMessages.join("\n"));
+            errorMessages.length = 0;
```

Comment:
this isn't conventional to override the length attribute of an array ^^ I don't think it's necessary anyway, since this callback is executed at the end of the suite (so nothing needs to be done afterwards)

Reply:
you can replace `const` by `let`